src/__tests__/__snapshots__/theme-provider.js.snap

```diff
@@ -127,11 +127,11 @@ exports[`with theme prop of padding 20px 1`] = `
       }
     }
   >
-    <glamorous(div)>
+    <glamorous.div>
       <div
         className="css-1e41o48"
       />
-    </glamorous(div)>
+    </glamorous.div>
```

Comment:
Yeah, maybe I'll hold off on this change until we're going to make a major version bump anyway :+1: